branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](http://localhost:9721/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=2&issues=AWU6UGSVcMk10FtshGc2&open=AWU6UGSVcMk10FtshGc2)

[ISSUE_KEY]: AWU6UGSVcMk10FtshGc2